recipe/meta.yaml

```diff
@@ -1,6 +1,6 @@
 {% set name = "versioneer" %}
-{% set version = "0.28" %}
-{% set sha256 = "7175ca8e7bb4dd0e3c9779dd2745e5b4a6036304af3f5e50bd896f10196586d6" %}
+{% set version = "0.29" %}
+{% set sha256 = "5ab283b9857211d61b53318b7c792cf68e798e765ee17c27ade9f6c924235731" %}
```

Comment:
I would remove this line and write the value directly to `source/sha256`. Instead of 2 lines will be 1.

The line `fn: {{ name }}-{{ version }}.tar.gz` can be removed as a redundant/outdated

Reply:
dev and doc urls should be `https://github.com/python-versioneer/python-versioneer`